make/mt-config/mt-config.mk

```diff
@@ -186,6 +191,10 @@ XI_SRCDIRS += $(LIBXIVELY_SOURCE_DIR)/datastructures
 XI_SRCDIRS += $(LIBXIVELY_SOURCE_DIR)/mqtt/codec
 XI_SRCDIRS += $(LIBXIVELY_SOURCE_DIR)/mqtt/logic
 XI_SRCDIRS += $(LIBXIVELY_SOURCE_DIR)/control_topic
+XI_SRCDIRS += $(LIBXIVELY_SOURCE_DIR)/cbor
+XI_SRCDIRS += $(LIBXIVELY_SOURCE_DIR)/cbor/cn-cbor/src
+# XI_SRCDIRS += $(LIBXIVELY_SOURCE_DIR)/cbor/tinycbor/src
```

Comment:
"TinyCBOR is Intel's industrial strength C/C++ implementation of CBOR, as used in the IoTivity framework:"
http://cbor.io/impls.html

I tried another cbor impl because the cn-cbor had some porting issues. But turned out that the other impls have porting issues as well. So I chose cn-cbor (Alan's choice).

And good catch, I remove this line from makefile!